lms/templates/ccx/schedule.underscore

```diff
@@ -26,7 +26,7 @@
           <tr class="sequential collapsed" data-depth="2"
               data-location="<%= chapter.location %> <%= child.location %>">
             <td class="unit">
-              <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
+              <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
```

Comment:
More importantly, they have no text value.  At the very least you should have screen reader text that describes the control "Toggle chapter display" with an aria-expanded attribute that describes the current state (true or false)